js/src/volume.js

```diff
@@ -246,6 +246,9 @@ var ScatterView = widgets.WidgetView.extend( {
                 animation_time_vz : { type: "f", value: 1. },
                 animation_time_size : { type: "f", value: 1. },
                 animation_time_color : { type: "f", value: 1. },
+                animation_time_index : { type: "f", value: 1. },
```

Comment:
No need for this, this should be reflected in the shaders js/glsl/scatter_vertex.js. But we can mimic a change in 'x', so just using animation_time_x, animation_time_y, animation_time_z.